packages/trader/src/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -2,7 +2,16 @@ import { PropTypes as MobxPropTypes } from 'mobx-react';
 import PropTypes from 'prop-types';
 import React from 'react';
 import { withRouter } from 'react-router-dom';
-import { DesktopWrapper, MobileWrapper, ProgressBar, Tabs, DataList, DataTable, ContractCard } from '@deriv/components';
+import {
+    DesktopWrapper,
+    MobileWrapper,
```

Comment:
Hmm what would be the advantage of this? :thinking: 
I think It's more readable to handle this with `jsx` instead of `js`. IMO we've created these wrappers to use while on a `jsx` context to improve readability. They're nothing more than a wrapper per the names suggest. Look at this:
```
const Desktop = ({ children }) => {
    if (!isDesktop()) return null;
    return children;
};
```
Just a `jsx` to cover the `js` behind it.

Reply:
Both are the same the only advantage of use isMobile is to decrease the number of statements. you can check both situations on one if statement both MobileWrapper and DesktopWrapper check contents separately